engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1142,6 +1142,17 @@ class PlatformDispatcher {
   bool get supportsShowingSystemContextMenu => _supportsShowingSystemContextMenu;
   bool _supportsShowingSystemContextMenu = false;
 
+  /// A callback that is invoked whenever [supportsShowingSystemContextMenu]
+  /// changes.
+  VoidCallback? get onSupportsShowingSystemContextMenuChanged =>
+      _onSupportsShowingSystemContextMenuChanged;
+  VoidCallback? _onSupportsShowingSystemContextMenuChanged;
+  Zone _onSupportsShowingSystemContextMenuChangedZone = Zone.root;
+  set onSupportsShowingSystemContextMenuChanged(VoidCallback? callback) {
+    _onSupportsShowingSystemContextMenuChanged = callback;
+    _onSupportsShowingSystemContextMenuChangedZone = Zone.current;
```

Comment:
Why was the analyzer complaining about the `Zone` related code removed from the `web_ui/lib/src/engine/platform_dispatcher.dart`?

Reply:
It complained that it was unused. I had just copied it from the similar code for textScaler or something.